src/contracts/bodhi_token.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import Contract from 'qweb3/src/contract';
+import Contract from 'qweb3/dist/contract';
```

Comment:
No, actually we need to update qweb3 to module.exports contract in index.js. That way this line's syntax becomes "import {Contract} from 'qweb3'" and need to change file path any more.

Reply:
nice, done! I noob at JS